app/presenters/vendor_api/application_presenter/add_sent_to_provider_datetime.rb

```diff
@@ -2,7 +2,7 @@ module VendorAPI::ApplicationPresenter::AddSentToProviderDatetime
   def schema
     super.deep_merge!({
       attributes: {
-        sent_to_provider_at: application_choice.sent_to_provider_at.iso8601,
+        sent_to_provider_at: application_choice.sent_to_provider_at&.iso8601,
```

Comment:
I don't think this is needed 🤔, we should always return submitted apps in Vendor API. Is there any on our DB?

Reply:
I fully agree but we've somehow triggered a Sentry error 🤷‍♂️

Reply:
I see what happened. We have the row in Support interface to see how the JSON Vendor API would be returned for that application choice. It makes sense.